jquery.passwd_field.js

```diff
@@ -0,0 +1,78 @@
+/*
+ * Password Visibility Toggle JQuery Plugin
+ *
+ * Adds password visibility toggle to a password field,
+ * by creating a another (text) input field. And switches
+ * between the password and the text input fields.
+ */
+(function ($) {
+
+    $.fn.passwordToggle = function () {
+
+        var docFrag = document.createDocumentFragment();
+        var random = [Math.floor(Math.random() * 9999)];
+        var randomCheckBoxId = 'data-psswrd-id-' + random;
+
+        var passwordElem = this[0];
+        passwordElem.classList.add('visiblity-toggle');
+        if (passwordElem === undefined) {
```

Comment:
This check is redundant.

The code is already accessing passwordElem.classList. This code is probably just left over?